### PR TITLE
fix: Adding a fix for copy clipboard URL not working on HTTP domain

### LINKED ISSUE #20574: [Bug]: share the application ,click "copy application URL",but no action!
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Description

when i share the application ,and click "copy application URL",but copy nothing
图片

### Steps To Reproduce

1.Click share on the top right corner
2. check "Make application public" to enable
3. click “copy application URL”
4. the system show no action!

### Public Sample App

_No response_

### Issue video log

_No response_

### Version

1.9.6,1.9.7

### PR DESCRIPTION
**Pull request by ankitakinger**
## Description

> Adding a fix for copy clipboard URL not working on HTTP domain.
> Adding Javascript origin and redirect URLs on the google auth settings page for better UX.
> Removing the upgrade button on the Appsmith watermark setting.
> Updating the placeholder for search input on members page.

Fixes #20574 #21170 

## Type of change

- Bug fix (non-breaking change which fixes an issue)
- Chore (housekeeping or task changes that don't impact user perception)

- Manual

### Dev activity
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [ ] PR is being merged under a feature flag

### QA activity:
- [ ] Test plan has been approved by relevant developers
- [ ] Test plan has been peer reviewed by QA
- [ ] Cypress test cases have been added and approved by either SDET or manual QA
- [ ] Organized project review call with relevant stakeholders after Round 1/2 of QA
- [ ] Added Test Plan Approved label after reveiwing all Cypress test

### CODE REVIEW

File: app/client/src/ce/constants/messages.ts [hunk 1]
```diff
@@ -1374,6 +1374,10 @@ export const MEMBERS_TAB_TITLE = (
   // eslint-disable-next-line @typescript-eslint/no-unused-vars
   cloudHosting?: boolean,
 ) => `Users (${length})`;
+export const SEARCH_USERS = (
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
+  cloudHosting?: boolean,
```
Comment (berzerkeer) [inline app/client/src/ce/constants/messages.ts:1379]: We don't need the param if we are not using it right. I believe we can remove the param.

**Review by berzerkeer — APPROVED** [review]

File: app/client/src/ce/constants/messages.ts (hunk 1 above)
Comment (ankitakinger) [reply app/client/src/ce/constants/messages.ts:1379]: Will be using it in EE

Comment (berzerkeer) [reply app/client/src/ce/constants/messages.ts:1379]: Got it 👍

**Pull request merged**